### ISSUE TITLE
regular segfaults, platform amd64- netbsd(-current) . (otter-browser version 9.0.9 from pkgsrc-current)

### ISSUE THREAD

**Issue by bu-ko-vsky** [OtterBrowser/otter-browser#1167]
I tried both precompiled packages for netbsd(with all "official" qt5-depending packages too!)... and also I tried to compile myself - all works just fine! Until i try to load (any, including local!) page.... then it (always!) segfaults... 
Important info - previous version (9.0.8) - din't work et all! At least in my case... (Of course I've already tried explicit disabling netbsd PaX mprotect+Segvguard+aslr features -  unfortunately results are exactly the same... ) If I can help the project, providing any details - feel free to let me know about it! I 'm very interested in project progress, cause I would really like to see this browser actually working! (exactly - rendering pages, since It is already works itself.... even on netbsd...and my opinion - it looks fantastic!!... but I'm not able to actually browse the web with it , yet....).. 
P.S. I also would like to know if it is already can work, (I mean really stable!) on some sort of linux distributions, maybe.... (or *bsd too! maybe there are some kind of "special instructions" how to make It work well on bsd... or just about on any free/gpl'ed OS- then I'll be able at least to install this OS/distro especially for browsing web, since all other "modern" browsers, that I tried - are total shit.. at this moment(... ))))

**Comment by Emdek** [comment]
@bu-ko-vsky, you should try some less ancient release first. ;-)
Which At version are you using? Constant crashes aren't something normal.
Any backtraces?

**Comment by mariokamenjak** [comment]
It works well for me on Linux. But if you want to sue it on Debian "Testing", you will have to use this workaround:
http://thedndsanctuary.eu/index.php?topic=2395.0

**Comment by Frenzie** [comment]
Actually I think the virtual package qtbase-abi-5-3-2 is in jessie now. :)

**Comment by mariokamenjak** [comment]
Yeah but it's not in stretch. It's in both jessie(stable) and sid(unstable) though.

**Comment by Frenzie** [comment]
Whoops, of course I mostly use stretch. XD still, it should come down from
sid soon enough.

**Comment by Emdek** [comment]
@bu-ko-vsky, package got updated, could you check again?
http://mail-index.netbsd.org/pkgsrc-changes/2016/08/20/msg145407.html

BTW, I recall that such issues can be caused by *libproxy* (Qt started using it AFAIR around 5.5), if you have KDE4 installed then it might pull in Qt4 which will cause crash (applies to every Qt5 application which uses system proxy for networking). 
In some cases it's possible to remove *libproxy-kde* or similar package, at least some Linux distributions allow to do that.

**Comment by Emdek** [comment]
Closing due to inactivity, please reopen if issue still persists.

**Comment by bu-ko-vsky** [comment]
Thanks for the tips guys! I check out the new version today,and if it will also crash, I'll give  you backtraces!

**Comment by bu-ko-vsky** [comment]
I tried new package version on netbsd! I'm installed it from precompiled packages at this point... And, yes, (I'm really surprised)), It doesn't crash so regullary no more... Local html-pages works just fine.
Remote web-sites sometimes also work))) sometimes not...(google etc... probably about ~70% of modern sites doesn't work, due to "incorrect request" or similar errors...) I have no idea about why it's happening and how to fix it... There may be many reasons for that((
But I think you shouldn't much worry about it... The things are getting better anyway... and I believe It will work well on netbsd soon. (Also, I may have just "incorrect" installation of any of the dependent packages.... Actually I don't thinks so, but It may be the case however... I'll try to figure out it in the near future).
AND I just tested otter-browser (0.9.11 weekly ??? version) on OpenBSD (6.0-stable/i386)... And It works quite well for me...
P.S. more ancient releases... It's not a suitable option for me(since I'd like to have some very new netbsd features/changes, available only in current as far as I know). More ancient releases of otter-browser I already tried, and It dind't work at all... But I can simply use openbsd with it's otter-browser package for surfing... NetBSD is my favorite system, but I don't use it a lot for web-browsing... 
That's ok. And furthermore, as a last resort, I'm thinking about installing linux especially for that reason... I have enough free space, and also I have debian-jessie distribution(only first dvd... but I think it should be pretty easy anyway to install otter on it/and get it work fine..)
Anyway thanks for all comments!
